src/main/java/com/palantir/weblogger/AnalyticsAppenderFactory.java

```diff
@@ -59,27 +65,27 @@
         appender.setPrudent(false);
 
         LayoutWrappingEncoder<ILoggingEvent> layoutEncoder = new LayoutWrappingEncoder<>();
-        layoutEncoder.setLayout(layout == null ? buildLayout(context, getTimeZone()) : layout);
+        layoutEncoder.setLayout(buildLayout(context, layoutFactory));
         appender.setEncoder(layoutEncoder);
 
         // instead of adding a threshold filter, filter out all events not directed at the analytics logger
         appender.addFilter(new AnalyticsFilter());
-        appender.stop();
```

Comment:
@zoubeiri logback RollingFileAppender changed its impl, and `stop()` is not needed anymore.
https://github.com/qos-ch/logback/commit/1af14db4b6f28aee79f8f1bc866dfffe6cfdbe5a#diff-168f8cbb3bd8b03acc60eb147007d385R51